lib/graphql/schema/input_object.rb

```diff
@@ -54,24 +62,12 @@ def initialize(arguments = nil, ruby_kwargs: nil, context:, defaults_used:)
         @maybe_lazies = maybe_lazies
       end
 
-      # @return [GraphQL::Query::Context] The context for this query
-      attr_reader :context
-
-      # @return [GraphQL::Query::Arguments, GraphQL::Execution::Interpereter::Arguments] The underlying arguments instance
-      attr_reader :arguments
-
-      # Ruby-like hash behaviors, read-only
-      def_delegators :@ruby_style_hash, :keys, :values, :each, :map, :any?, :empty?
-
       def to_h
-        @ruby_style_hash.inject({}) do |h, (key, value)|
-          h.merge(key => unwrap_value(value))
+        @ruby_style_hash.reduce({}) do |h, (key, value)|
+          h.merge!(key => unwrap_value(value))
         end
```

Comment:
Interesting... it looks like the implementation here is the same as `unwrap_value` below. (It was the same before, too.) What do you think of DRYing it? 

```suggestion
        unwrap_value(@ruby_style_hash)
```

Reply:
Makes sense 🙂